src/main/java/net/technolords/micro/model/jaxb/resource/SimpleResource.java

```diff
@@ -12,6 +12,22 @@
     private int errorRate;
     private String contentType;
 
+    @Override
+    public boolean equals(Object obj) {
+        if (obj == null) {
+            return false;
+        }
+        if (!(obj instanceof SimpleResource)) {
+            return false;
+        }
+        SimpleResource ref = (SimpleResource) obj;
+        return (ref.getResource().equals(this.getResource())
+//                && ref.getCachedData().equals(this.getCachedData())
```

Comment:
Why commenting these out? It should be complete if these are included in the equals.